pom.xml

```diff
@@ -62,7 +62,7 @@
         <version.io.kaitai>0.10</version.io.kaitai>
 
         <!-- this is the class lib version; the class lib cannot be used in a POM but is used in resources -->
-        <version.qbicc.classlib>17.alpha.0.43</version.qbicc.classlib>
+        <version.qbicc.classlib>17.alpha.0.45-SNAPSHOT</version.qbicc.classlib>
```

Comment:
This looks like the reason CI is failing.

Reply:
fixed...too much multitasking.